filters/regional/filter_54_AWAvenue-Adblock-RuleRule/filter.txt

```diff
@@ -0,0 +1,362 @@
+[-AWAvenue 秋风广告规则（AWAvenue-Adblock-Rule）-]

```

Comment:
Only 2 files should be created in the folder: configuration.json and metadata.json

Reply:
Deleted as requested, please review, thank you！